pkg/deps/types/accquerier.go

```diff
@@ -0,0 +1,53 @@
+package types
+
+import (
+	"context"
+
+	"cosmossdk.io/depinject"
+	"github.com/cosmos/cosmos-sdk/client"
+	accounttypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+
+	"github.com/pokt-network/poktroll/pkg/relayer"
+)
+
+type AccountQuerier interface {
```

Comment:
I :heart: that you've introduced an interface here! This is great because it more concisely represents the **shared/common** needs of **our** pkgs. :mechanical_arm:  It also, of course, provides better decoupling from cosmos-sdk, which I think is something we should prefer, especially when it's convenient in the moment. This is an early (and compounding) investment in our pkgs being able to support non-cosmos chains. :rocket:

That being said, I feel like this interface probably makes more sense to live in pkg/client/interface.go. This is a real thing describing real behavior, let's categorize it as such. :sunglasses: